### DISCUSSION TITLE
Problem with sms_adapter PCB

### DISCUSSION THREAD

**Discussion by Akamaru** [sanni/cartreader#100] [Need help]
Hi,
I ordered the sms_adapter PCB at PCBWay. Today I got a mail.

> Thanks for your order. There is an engineering question of order XXX.
> 1. Pls find picture 1, spacing between grids is only 0.1524mm,  it is too  narrow.
> Pls kindly check and advise if we can fill in grids and make as copper plane. If you would like us to make as per your file, we will use wet file to produce.
> 2. Pls find picture 2, part of gold fingers are without soldermask openings, they will be covered with soldermask ink.
> Pls kindly advise if we should make as per your file or make soldermask openings for whole gold fingers.
> 
> Waiting for your comments to proceed.
> Pls note, the delivery time will be recalculated by the end of EQ.

![EQ of W395892AS1Y8(1)](https://user-images.githubusercontent.com/8063440/108706690-422aea00-750f-11eb-9dcf-0a26578ae466.png)
![EQ of W395892AS1Y8(2)](https://user-images.githubusercontent.com/8063440/108706694-42c38080-750f-11eb-83c0-9698295268b8.png)

What should I do?

**Comment by sanni** [comment]
1. fill in grids and make as copper plane
2. make as per file

**Reply by Akamaru** [reply]
Okay, thank you!
